contracts/Account.sol

```diff
@@ -573,6 +679,74 @@ contract Account is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed, I
         return scheduledVotes[group].toWithdrawFor[beneficiary];
     }
 
+    /**
+     * @notice Returns the storage, major, minor, and patch version of the contract.
+     * @return Storage version of the contract.
+     * @return Major version of the contract.
+     * @return Minor version of the contract.
+     * @return Patch version of the contract.
+     */
+    function getVersionNumber()
+        external
+        pure
+        returns (
+            uint256,
+            uint256,
+            uint256,
+            uint256
+        )
+    {
+        return (1, 2, 0, 0);
+    }
+
+    /**
+     * @notice Revokes votes from a validator group. It first attempts to revoke pending votes,
+     * and then active votes if necessary.
+     * @dev Reverts if `revokeAmount` exceeds the total number of pending and active votes for
+     * the group from this contract.
+     * @param group The group to revoke CELO from.
+     * @param lesserAfterPendingRevoke Used by Election's `revokePending`. This is the group that
+     * is before `group` within the validators sorted LinkedList, or address(0) if there isn't one,
```

Comment:
Correct me if i'm wrong.

If currentVotes are sorted from greatest to lowest value, shouldn't lesserAfterPendingRevoke be "the group that is **after** `group`..."

Reply:
I believe that it is sorted from lowest to greatest. Tail (on far left)  is lowest and Head is greatest (on far right).

Reply:
On second thought I dont believe there is any orientation in sense before/after. There is only concept of lesser and greater. I actually took part of  this comment from other functions (so if we would agree to change it we will have to update it also there). But since usually when ordered it is considered ascending I would probably keep it as it is. Or we could rephrase and omit any before/after and rather use lesser and greater.